code/modules/plumbing/ducts.dm

```diff
@@ -205,6 +204,9 @@ All the important duct code:
 /obj/machinery/duct/proc/add_connects(new_connects) //make this a define to cut proc calls?
 	if(!lock_connects)
 		connects |= new_connects
+/obj/machinery/duct/proc/remove_connects(dead_connects)
```

Comment:
documentation

Reply:
And a newline above while at it

Reply:
You need `///` for autodocs and this still doesn't have a newline